Plots/MapProjections/NCL_lcnative_1.py

```diff
@@ -1,6 +1,6 @@
 """
 NCL_lcnative_1.py
-=================
```

Comment:
Sphinx will throw a warning with this saying that the title underline is too short. Make sure the underline is the same length as the title.